src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -410,7 +438,144 @@ lemma tendsto_test_against_nn_of_le_const {μ : finite_measure α}
 tendsto_test_against_nn_filter_of_le_const
   (eventually_of_forall (λ n, eventually_of_forall (fs_le_const n))) (eventually_of_forall fs_lim)
 
-end finite_measure
+end finite_measure -- namespace
+
+end finite_measure_bounded_convergence -- section
+
+section finite_measure_convergence_by_bounded_continuous_functions
+/-! ### Weak convergence of finite measures with bounded continuous real-valued functions
+
+In this section we characterize the weak convergence of finite measures by the most common
+(defining) condition of convergence of integrals of all bounded continuous real-valued functions.
+-/
+
+namespace finite_measure
+
+variables {α : Type*} [measurable_space α] [topological_space α] [opens_measurable_space α]
+
+lemma integrable_of_bounded_continuous_to_nnreal
+  (μ : measure α) [is_finite_measure μ] (f : α →ᵇ ℝ≥0) :
+  integrable ((coe : ℝ≥0 → ℝ) ∘ ⇑f) (μ : measure α) :=
+begin
+  refine ⟨(nnreal.continuous_coe.comp f.continuous).measurable.ae_strongly_measurable, _⟩,
+  simp only [has_finite_integral, nnreal.nnnorm_eq],
+  exact lintegral_lt_top_of_bounded_continuous_to_nnreal _ f,
+end
+
+lemma integrable_of_bounded_continuous_to_nnreal' (μ : finite_measure α) (f : α →ᵇ ℝ≥0) :
+  integrable ((coe : ℝ≥0 → ℝ) ∘ ⇑f) (μ : measure α) :=
```

Comment:
I don't see the point of this lemma: it is just a specialization of the previous one as Lean can find that the coercion of a `finite_measure` satisfies `[is_finite_measure ...]`. Indeed, the proof `integrable_of_bounded_continuous_to_nnreal _ _` just works.

Reply:
Sorry! You are right, of course.

This was left when I changed to the current formulation. I "temporarily" left the old formulation as a primed version while making sure that I can fix all proofs with the new formulation. But then forgot to remove the "temporarily left" old formulation.